tests/robustness/failpoint/trigger.go

```diff
@@ -78,11 +78,21 @@ func (t triggerCompact) Trigger(ctx context.Context, _ *testing.T, member e2e.Et
 	}
 	_, err = cc.Compact(ctx, rev)
 	if err != nil && !strings.Contains(err.Error(), "error reading from server: EOF") {
-		return nil, err
+		return nil, fmt.Errorf("failed to compact: %w", err)
 	}
 	return []report.ClientReport{cc.Report()}, nil
 }
 
-func (t triggerCompact) Available(e2e.EtcdProcessClusterConfig, e2e.EtcdProcess) bool {
+func (t triggerCompact) Available(config e2e.EtcdProcessClusterConfig, _ e2e.EtcdProcess) bool {
+	// Since introduction of compaction into traffic, injecting compaction failpoints started interfeering with peer proxy.
+	// TODO: Re-enable the peer proxy for compact failpoints when we confirm the root cause.
```

Comment:
Is there an issue tracking this?

Reply:
Not yet, I think https://github.com/etcd-io/etcd/pull/16788 might be related. 

I was also hoping that maybe this will be auto resolved with https://github.com/etcd-io/etcd/pull/17938 as major rewrite might change.